### ISSUE TITLE
Is there any way to connect to oracle database?

### ISSUE THREAD

**Issue by guicovred** [mikro-orm/mikro-orm#810]
Hi, 

Is there a way to use MikroORM with oracle databases? if not, is there anything on the road map for that?

Thanks!

**Comment by B4nan** [comment]
Nope, no driver for oracle yet, you would have to create the driver yourself (but if knex supports it, it should not be too much work). No plans to work on this in near future (4.x), but if there would be some initial work done I would be happy to finalize it and add it to natively supported drivers. That currently happens with mssql - there was some initial work done, and now I am building on top of that. 

In general the first step for integration would be to setup local environment (so docker). 

Will be happy to help, ideally join the slack channel and we can discuss it there.

**Comment by guicovred** [comment]
I made a pull request with some initial work to add oracle support. Still lacks automated testing and some other things, but I will keep trying to help bring this functionality to mikro-orm. In some parts i'm not sure if i understood what the behavior should be, but i tried not to bother you for now.

For the local development environment, I used this [Oracle database docker image](https://hub.docker.com/_/oracle-database-enterprise-edition), since oracle allows its use for development purposes ([Terms of use](https://www.oracle.com/downloads/licenses/standard-license.html)).

I joined the slack channel. Thanks for your attention!
